### PR TITLE
fix for google https

### PR DESCRIPTION
**Pull request by ElleryHorton**
Not sure why the http / https difference didn't fail previously but now it did.

### CODE REVIEW

File: Zukini.UI.Examples.Features/Steps/SmokeTestSteps.cs [hunk 1]
```diff
@@ -135,5 +135,41 @@ public void ThenTheSecondButtonShouldExist()
...
+            {
+                Browser.WaitForNavigation(_sessionConfiguration, url);
+                return false;
+            }
+            catch (TimeoutException)
+            {
+                return true;
+            }
+        }
+
```
Comment (MikeMugu) [inline Zukini.UI.Examples.Features/Steps/SmokeTestSteps.cs]: Can you move this to the bottom of the class? Just to separate it as a private method in the class from a step def implementation.

File: Zukini.UI.Examples.Features/App.config [hunk 2]
```diff
@@ -12,8 +12,9 @@
     <add key="MaximizeBrowser" value="true" />
     <add key="ScreenshotDirectory" value="..\..\..\Screenshots" />
     <!-- Application Under Test Settings -->
-    <add key="GoogleUrl" value="http://www.google.com" />
-    <add key="W3SchoolsBaseUrl" value="http://www.w3schools.com" />
+    <add key="GoogleUrl" value="https://www.google.com" />
+    <add key="GoogleHttpUrl" value="http://www.google.com" />
+    <add key="W3SchoolsBaseUrl" value="https://www.w3schools.com" />
     <add key="JsonPlaceholderApiUrl" value="http://jsonplaceholder.typicode.com" />
```
Comment (MikeMugu) [inline Zukini.UI.Examples.Features/App.config:18]: Would we still need both settings or are we always going to use HTTPS now?

File: Zukini.UI.Examples.Features/Steps/SmokeTestSteps.cs (hunk 1 above)
Comment (ElleryHorton) [reply Zukini.UI.Examples.Features/Steps/SmokeTestSteps.cs]: done

File: Zukini.UI.Examples.Features/App.config (hunk 2 above)
Comment (ElleryHorton) [reply Zukini.UI.Examples.Features/App.config:18]: The HTTP setting is to verify that WaitForNavigation does timeout (negative test). Should always use HTTPS setting for other tests. Could remove the negative test as an alternative.

**Review by MikeMugu — APPROVED** [review]

**Pull request merged**